docs/_posts/api/0200-01-01-v2.2.1-all.html

```diff
@@ -151,7 +151,7 @@ <h2 id="l-mapbox-gridlayer">L.mapbox.gridLayer(id|url|tilejson, options)</h2>
 </table>
 <p><em>Example</em>:</p>
 <pre><code>// the second argument is optional
-var layer = L.mapbox.gridLayer(&#39;mapbox.streets&#39;);
+var layer = L.mapbox.gridLayer(&#39;examples.map-20v6611k&#39;);
```

Comment:
Reading through, these docs need some additional love w/r/t v3 and `examples`. I'm going to fix these instances and merge, but then go through and do some additional cleanup in a different PR.